python/packages/devui/agent_framework_devui/_server.py

```diff
@@ -480,34 +480,53 @@ async def get_entity_info(entity_id: str) -> EntityInfo:
                         workflow_dump = {k: v for k, v in entity_obj.__dict__.items() if not k.startswith("_")}
 
                     # Get input schema information
-                    input_schema = {}
+                    # First, check if the workflow has an explicit input_schema (e.g., declarative workflows)
+                    input_schema: dict[str, Any] = {}
```

Comment:
@victordibia I had to do some changes here to get declarative workflows to parse properly... can you have a look please? Not sure how it'll jive with your current DevUI PR you have open. Thanks.